packages/expo-media-library/android/src/main/java/expo/modules/medialibrary/MediaLibraryModule.kt

```diff
@@ -490,7 +494,7 @@ class MediaLibraryModule : Module() {
     private fun getAssetsTotalCount(mediaType: Int): Int =
       context.contentResolver.query(
         EXTERNAL_CONTENT_URI,
-        null,
+        arrayOf(),
```

Comment:
https://developer.android.com/reference/android/content/ContentResolver#query(android.net.Uri,%20java.lang.String[],%20java.lang.String,%20java.lang.String[],%20java.lang.String,%20android.os.CancellationSignal)

> String: A list of which columns to return. Passing null will return all columns, which is inefficient.


here we don't care about any of the columns, we just care about the number of items in the cursor